### DISCUSSION TITLE
Github Student Developer Pack application getting rejected again and again.

### DISCUSSION THREAD

**Discussion by shadow-dragon-2002** [community/community#57348] [GitHub Education] [answered]
### Hi everyone,

Hello everyone, 

For more details, I am currently a distance learner student from Manipal University Jaipur and all the documents they provided me i.e. the ID card, fee receipts, etc. Everything is in my email and is in pdf format and if I try to change it to jpg and upload it it rejects it giving the following feedback :
___________________________________________________________________________________________________________________________________________
Unfortunately, we weren't able to approve your educational discount request for [*******].

What happened?
The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.
____________________________________________________________________________________________________________________________________________
I tried changing and uploading my college ID and a fee receipt for one of my semesters but both got rejected. I do have a university-provided email id which I did use but GitHub said I require submitting further proof. I am not sure what further to do as there is no way to send the pdf directly to any GitHub support, is there any solution for this? Thank you very much in advance.
____________________________________________________________________________________________________________________________________________
### Alright finally got it

So I had a 2-page ID which was the problem but instead of editing it manually I put it through an image merger online merged it vertically and as dakshdeepHERE suggested in the latest reply changed my name to my official name on GitHub, then I resubmitted the application and it worked

**Comment by dakshdeepHERE** [comment]
@shadow-dragon-2002 Same thing happened with me before I did used my ID card first. It got rejected and I did my research and came to know that its all done by bots so if your pic is not clear and if the bot cannot confirm the college name, your name and your graduation date then your request will automatically will be rejected and same thing was happening with me. I even linked my time table my course structure but nothing worked. In the end I went to the college page and there the website had a virtual id card with was perfect. I took the ss of it and linked everything in the mail. Boom I got my student profile verified easily. Good luck try this method if it works for u.

**Reply by shadow-dragon-2002** [reply]
Thanks for the suggestion but the virtual ID my university provided me basically is has 2 pages instead of like a normal id card. The name and picture is on the first page and the student till date os the second page so that's where that problem for me arose.

**Reply by dakshdeepHERE** [reply]
Then don't attach virtual id only attach the alternatives like time table and course structures they also have your name and other infos

**Reply by dakshdeepHERE** [reply]
> Maybe a little late to the party but in some cases your bio might be too short.

Wow, this can be a case too?

**Reply by dakshdeepHERE** [reply]
So true plus trying to connect support on this issue is also really tough.

**Comment by dakshdeepHERE** [comment, marked as answer]
> You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

See its not able to confirm your name on the documents and github profile are matching or not.

**Reply by shadow-dragon-2002** [reply]
Ah alright, I'll try putting my GitHub name as my official name and see if that works thank you for the suggestion.

**Reply by dakshdeepHERE** [reply]
Sure go on

**Reply by shadow-dragon-2002** [reply]
Alright thank you for the help man I changed my name and uploaded the ID again but instead of manually editing and merging it I put it through an online merger I found and it worked. So Thank you very much you suggestions helped me quite a bit.

**Reply by dakshdeepHERE** [reply]
Hey @shadow-dragon-2002, I'm really delighted to hear that your issue has been fixed! 😃

**Comment by dakshdeepHERE** [comment]
> I tried changing and uploading my college ID and a fee receipt for one of my semesters but both got rejected. I do have a university-provided email id which I did use but GitHub said I require submitting further proof. I am not sure what further to do as there is no way to send the pdf directly to any GitHub support, is there any solution for this? Thank you very much in advance.

Add your college mail as an alternative mail and keep the primary mail from which you are mailing github.

**Reply by marifcelik** [reply]
Hi, which e-mail address did you use? I opened a ticket, but I couldn't find an e-mail address to send an e-mail to.

**Reply by shadow-dragon-2002** [reply]
I did add my college mail as an alternative. It's an edu.in email address and it is added to my GitHub profile as a secondary email, and that is what I used to apply.
